### DISCUSSION TITLE
Daily Build

### DISCUSSION THREAD

**Discussion by TheTubaGeek** [Brewtarget/brewtarget#648] [Q&A] [answered]
I was told that I can download the nightly build from GitHub, but I am having problems locating it. Can someone point me in the right direction?

I saw an MSI for version 2.4.0, so I'm downloading and installing it to see if it fixes the issue I was having with the version I had installed.

**Comment by TheTubaGeek** [comment, marked as answer]
UPDATE: 2.4.0 allowed me to import from some BEERXML files I found online without issue. In fact, one of the files had a lot of errors and thanks to the alerts and Notepad++, I was able to edit the file in question.

I am providing the files here in case anyone would like to use them. Just rename the extension:

[grain.txt](https://github.com/Brewtarget/brewtarget/files/8535862/grain.txt)
[hops.txt](https://github.com/Brewtarget/brewtarget/files/8535863/hops.txt)
[misc.txt](https://github.com/Brewtarget/brewtarget/files/8535865/misc.txt)
[style.txt](https://github.com/Brewtarget/brewtarget/files/8535866/style.txt)
[yeast.txt](https://github.com/Brewtarget/brewtarget/files/8535867/yeast.txt)

**Reply by matty0ung** [reply]
Glad to hear this was useful.  Gradually we want to update the https://github.com/Brewtarget/brewtarget/wiki to have more useful info, especially about getting started with the software.

**Comment by TheTubaGeek** [comment]
Yes, especially since this new version has features that the previous
version does not.

[image: created with MySignature.io] <https://mysig.io/ZjJjOTV>
Russell Wright

Jr. Business Systems Analyst | FIS Global | Integration Services - PC
Products
M:  +1 704.963.0518 <+1+704.963.0518>
E:  ***@***.***
A:  Concord, NC
<https://mysignature.io/editor/?utm_source=freepixel>
[image: Create your own signature]
<https://mysignature.io/?utm_source=promotion&utm_medium=signature&utm_campaign=create_own_signature>



<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
